### PR TITLE
Add namespaces and fix s3 initdb

### LINKED ISSUE #1202: Fix review comments for multicloud-ha
**Issue/Feature Description:**
- IMHO Sensitive info like username and password should be stored in Secrets.
Ref: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables

- The mount can be ignore by using kubernetes logging pipeline

- we can apply multiple resources in a directory and sub directory by
kubectl apply -f <directory> --recursive

- I think its not good choice to provide additional privileges to default service account.
A good choice would be to create a namespace , add a service account and give access to that service account.
Also same Service account need to be get used for add deployment of application trying to access kubernetes APIs by github.com/micro/go-plugins/registry/kubernetes/v2

**Why this issue to fixed / feature is needed(give scenarios or use cases):**

**How to reproduce, in case of a bug:**

**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**

### PR DESCRIPTION
**Pull request by rajat-soda**
**What type of PR is this?**
-  Adds namespace in deployment files itself, and removed from the k8s-deploy.sh file. 
-  Fix initdb.sh to point to correct sql file. 
-  Made corresponding changes to docker-compose and Dockerfile of S3 service. 
-  Use Latest image to run the pods.
-  Make mongo scripts executable.
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:

Without the PR , S3 service does not comeup . and makes the s3 related features unusable. 

**Which issue(s) this PR fixes**:

Fixes #1202 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED

Created bucket, Uploaded object, Created folder 

image

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rajat-soda** [comment]
@AmitRoushan , @asifdxtreme @kumarashit : Please review and merge.

File: K8s/k8s-mongo.sh [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+kubectl apply -f mongo-pvc-0.yaml 
+kubectl apply -f mongo-pvc-1.yaml 
+kubectl apply -f mongo-pvc-2.yaml 
 
-kubectl apply -f mongo-service-0.yaml -n soda-multi-cloud
-kubectl apply -f mongo-service-1.yaml -n soda-multi-cloud
-kubectl apply -f mongo-service-2.yaml -n soda-multi-cloud
+kubectl apply -f mongo-service-0.yaml 
+kubectl apply -f mongo-service-1.yaml 
+kubectl apply -f mongo-service-2.yaml
```
Comment (AmitRoushan) [inline K8s/k8s-mongo.sh:15]: can we merge k8s-mongo.sh and k8s-deploy.sh files

Comment (rajat-soda) [reply K8s/k8s-mongo.sh:15]: No

Comment (rajat-soda) [reply K8s/k8s-mongo.sh:15]: Will do once stable..

**Review by AmitRoushan — CHANGES_REQUESTED** [review]
LGTM

**Review by AmitRoushan — APPROVED** [review]
LGTM

**Review by asifdxtreme — APPROVED** [review]
LGTM

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by NajmudheenCT — APPROVED** [review]
LGTM
Please change image tag configurable as discussed

**Pull request merged**